docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -14,6 +14,25 @@ additional driver and software components are properly installed and running.
 
 [Intel SGX]: https://www.intel.com/content/www/us/en/architecture-and-technology/software-guard-extensions.html
 
+## BIOS Configuration
+To set up a TEE (Intel SGX), you need to configure BIOS. First, **update BIOS 
+to the latest version with the latest microcode**. Then proceed with BIOS 
+configuration. Note that not all settings may apply to your BIOS. In that case, 
+configure only the relevant ones. Please set the BIOS settings as follows:
+
+- **SGX**: ENABLE
+- **Hyperthreading**: DISABLE
+- **Intel SpeedStep**: DISABLE
+- **SecureBoot**: DISABLE
```

Comment:
Why does SecureBoot need to be disabled?

Reply:
I do not know why, but it needs to be disabled. Once, we had a problem with the Cipher node and did not know what was wrong. Then community member shared that SecureBoot needs to be disabled. So we tried, and after that, it started working.

Reply:
Hm, ok maybe it had to do with loading a custom SGX driver which was unsigned (and secure boot requires signed drivers)? This should not be a problem for the mainline driver that is included in recent kernels.

Reply:
Added a comment that this setting is not necessary for recent kernels